scitt-emulator.sh

```diff
@@ -6,7 +6,7 @@ set -e
 
 if [ ! -f "venv/bin/activate" ]; then
     echo "Setting up Python virtual environment."
-    python3.8 -m venv "venv"
```

Comment:
This worked for me on Apple M1, default python version is higher, don't think we need the pin here.

Reply:
On Linux, `python` will still mostly give you Python 2 unfortunately. Does `python3` work on Apple?

Reply:
It did for me, after removing the pinning.

Reply:
We should have something that works both on Linux and macOS I think.

Reply:
Are those really the only 2 operating systems that matter? ; )

Reply:
Maybe there is a better way to fix the script, I am not really a "python" expert.

In node.js we use `nvm` and `npm` scripts for this kind of thing.

Reply:
yes, `python3` works, I updated the PR to use it.